tests/test_laser_profiles.py

```diff
@@ -41,6 +46,92 @@ def gaussian():
     return profile
 
 
+def test_transverse_profiles_rt():
+    npoints = 4000
+    w0 = 10.0e-6
+    r = np.linspace(0, 12 * w0, npoints)
+
+    # GaussianTransverseProfile
+    print("GaussianTransverseProfile")
+    std_th = w0 / np.sqrt(2)
+    profile = GaussianTransverseProfile(w0)
+    field = profile.evaluate(r, np.zeros_like(r))
+    std = np.sqrt(np.average(r**2, weights=np.abs(field)))
+    print("std_th = ", std_th)
+    print("std = ", std)
+    assert np.abs(std - std_th) / std_th < 0.01
+
+    # LaguerreGaussianTransverseProfile
+    print("LaguerreGaussianTransverseProfile")
+    p = 2
+    m = 0
+    std_th = np.sqrt(5 / 2) * w0
+    profile = LaguerreGaussianTransverseProfile(w0, p, m)
+    field = profile.evaluate(r, np.zeros_like(r))
+    std = np.sqrt(np.average(r**2, weights=r * np.abs(field) ** 2))
+    print("std_th = ", std_th)
+    print("std = ", std)
+    assert np.abs(std - std_th) / std_th < 0.01
+
+    # SuperGaussianTransverseProfile
+    print("SuperGaussianTransverseProfile")
+    n_order = 100  # close to flat-top, compared with flat-top theory
+    std_th = w0 / np.sqrt(3)
+    profile = SuperGaussianTransverseProfile(w0, n_order)
+    field = profile.evaluate(r, np.zeros_like(r))
+    std = np.sqrt(np.average(r**2, weights=np.abs(field)))
+    print("std_th = ", std_th)
+    print("std = ", std)
+    assert np.abs(std - std_th) / std_th < 0.01
+
+    # JincTransverseProfile
+    print("JincTransverseProfile")
+    profile = JincTransverseProfile(w0)
+    std_th = 1.5 * w0  # Just measured from this test
+    field = profile.evaluate(r, np.zeros_like(r))
+    std = np.sqrt(np.average(r**2, weights=abs(field) ** 2))
+    print("std_th = ", std_th)
+    print("std = ", std)
+    assert np.abs(std - std_th) / std_th < 0.1
+
+
+def test_transverse_profiles_3d():
+    npoints = 200
+    w0 = 10.0e-6
+
+    # HermiteGaussianTransverseProfile
+    print("HermiteGaussianTransverseProfile")
+    n_x = 2
+    n_y = 2
+    std_th = np.sqrt(5.0 / 4) * w0
+    profile = HermiteGaussianTransverseProfile(w0, n_x, n_y)
+    x = np.linspace(-4 * w0, 4 * w0, npoints)
+    y = np.zeros_like(x)
+    field = profile.evaluate(x, y)
+    std = np.sqrt(np.average(x**2, weights=np.abs(field) ** 2))
```

Comment:
I modified this (`np.abs(field) -> np.abs(field)**2`) so that it has an easy analytical solution